### PR TITLE
Fix issues to properly test

### PR DESCRIPTION
**Pull request by therubensan**

### CODE REVIEW

File: src/main/java/com/google/sps/data/TrickNode.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+// a class that holds data pertaining to a trick
+public class TrickNode {
+  public String trick_name;
+  public long date;
+  public String link;
+  public String notes;
```
Comment (CubeOfCheese) [inline src/main/java/com/google/sps/data/TrickNode.java:8]: Typically class attributes are made private and get/set methods are used for each attribute

File: src/main/webapp/hello.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8">
+    <title>Trick Tracker</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+     <h1>Hello World!</h1>
+     <button><a href="/timeline.html">Drop In</a></button>
+  </body>
+</html>
```
Comment (CubeOfCheese) [inline src/main/webapp/hello.html]: This file should be deleted

File: src/main/webapp/trickeditpage.html [hunk 3]
```diff
@@ -11,18 +11,21 @@
       <div class="form-header">
         <h2>Add Trick:</h2>
       </div>
-      <form style="margin:10px;input" action="/timeline_data" method="POST" onsubmit="return loadResults();">
+
+      <form style="margin:10px;input" action="/add_trick" method="POST" onsubmit="return loadResults();>
```
Comment (CubeOfCheese) [inline src/main/webapp/trickeditpage.html]: Change to /add-trick for consistency

File: src/main/java/com/google/sps/servlets/AddTrickServlet.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.text.ParsePosition;
+import java.text.SimpleDateFormat;  
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Locale;
+
+/** Servlet that adds tricks to a user's timeline / datastore */
+@WebServlet("/add_trick")
```
Comment (CubeOfCheese) [inline src/main/java/com/google/sps/servlets/AddTrickServlet.java]: /add-trick please

File: src/main/webapp/trickeditpage.html [hunk 5]
```diff
@@ -11,18 +11,21 @@
...
         <label for="skate-style">Skate style:</label><br>
         <input type="text" id="skate-style" name="skate-style" placeholder="Longboarding"><br>
-        
         <label for="trick-name">Trick name:</label><br>
         <input type="text" id="trick-name" name="trick-name" placeholder="Ollie"><br>
 
         <label for="date">Date:</label><br>
         <input type="text" id="date" name="date"><br>
```
Comment (CubeOfCheese) [inline src/main/webapp/trickeditpage.html]: Is there a reason for having two date fields? This one seems residual

**Review by CubeOfCheese — CHANGES_REQUESTED** [review]

File: src/main/webapp/hello.html (hunk 2 above)
Comment (therubensan) [reply src/main/webapp/hello.html]: The file should be deleted now. Let me know if it shows on your end.

**Review by CubeOfCheese — APPROVED** [review]

**Pull request merged**